packages/@core.keystone/test.utils.js

```diff
@@ -97,151 +113,176 @@ const prepareNextExpressApp = async (dir) => {
     return { app }
 }
 
-const makeFakeClient = async (app, server) => {
-    const request = require('supertest')
-    const port = server.address().port
-    const protocol = app instanceof https.Server ? 'https' : 'http'
-    const serverUrl = protocol + '://127.0.0.1:' + port
+/**
+ * @param {Error} e
+ * @param {Object[]} e.graphQLErrors
+ * @param {ServerError} e.networkError
+ * @param {String} e.message
+ * @param {*} e.extraInfo
+ * @param {String} e.stack
+ * @returns {{errors: []}}
+ */
+function gqlCatchedErrorsHandler (e) {
```

Comment:
why did you extract this function here? probably it's useless outside of the `makeApolloClient` context. Probably, it's better to define it as `restoreCookies`/`saveCookies`?

Reply:
done